commercetools-models/src/main/java/io/sphere/sdk/customers/Customer.java

```diff
@@ -128,7 +128,7 @@
         "StringQuerySortingModel<Customer> lowercaseEmail();",
         "BooleanQueryModel<Customer> isEmailVerified();"
 })
-@HasUpdateActions
+@HasUpdateActions(deriveExampleBaseClass = false)
```

Comment:
This reveals that our integration tests for the `CustomerUpdateCommand` is missing a few tests for some update actions. How about adding them and then we can remove the `deriveExampleBaseClass` here? 😉